.ls-lint.yml

```diff
@@ -30,3 +30,5 @@ ignore:
   - '**/dist'
   - '**/documentation'
   - '**/node_modules'
+  - './apps/doc_build'
```

Comment:
nit: for folders we control, stick to `kebab-case` instead of `snake_case` per our linting rules

Reply:
Agreed. This is the default for rspress.